### ISSUE TITLE
Columns default width

### ISSUE THREAD

**Issue by firedev** [jakob/Postico#295]
When using `uuid` as a primary keys the db becomes hard to look at.

![image](https://cloud.githubusercontent.com/assets/115612/20419306/3fe4f914-ad88-11e6-9350-96eb9dbef6db.png)

You can make columns more narrow and Postico remembers the width for the duration of the current session but then after another launch they are having full-width again. 

So I was thinking if storing view settings per each db might be not a viable option, maybe it is possible to add like default max-width for datatypes? After all, there is some trimming present as it is.

**Comment by jakob** [comment]
For performance reasons, Postico only looks at the row with the longest contents (in characters) to check the size. For text this works quite well, but for strings with a fixed length this doesn't work so well. I'll have to see if I can fix the autosize algorithm without sacrificing performance.

In the mean time, you could also try using the new monospace font preference that I just added; this would make working with UUID columns a lot more pleasant I suppose:
https://eggerapps-downloads.s3-eu-west-1.amazonaws.com/postico-1593.zip

**Comment by firedev** [comment]
Thanks, yes it is a bit better. Could we also change the font face? I am using [Iosevka](https://be5invis.github.io/Iosevka/) which is fairly narrow.

**Comment by jakob** [comment]
Preferences for default font face is definitely something I'm considering (together with themes / configurable styles for syntax highlighting) -- but I really want to finish a few more important features first.

**Comment by jakob** [comment]
Hey, I just found out that there is a hidden setting to change the default fixed width font in macOS.

You can use this command to change the monospace font:

```bash
defaults write at.eggerapps.Postico NSFixedPitchFont "Courier"
```

After  restart, Postico should sport the new font!

**Comment by firedev** [comment]
Wow, this is pretty sweet, thanks!

![image](https://cloud.githubusercontent.com/assets/115612/23005325/71da1bce-f42e-11e6-92e2-019eba200909.png)

**Comment by jakob** [comment]
Autosizing of ENUM columns is now fixed in Postico 1.5.15
